Source/CombatExtended/CombatExtended/CE_Utility.cs

```diff
@@ -14,6 +14,17 @@ namespace CombatExtended
 {
     static class CE_Utility
     {
+        #region Camera
+
+        public static float CameraAltitude
```

Comment:
Is this used anywhere?

Reply:
It'll be used since occlusion for flecks is an issue when zooming in. I would just leave it there since i have a prototype that use it for flecks with Y axis support.